did-sdk/src/web/web.ts

```diff
@@ -2,17 +2,25 @@ import { DIDDocument, parse } from 'did-resolver'
 import fetch from 'node-fetch'
 import { ReadOnlyDidDocAPI } from '../did-documents.js'
 
-const DOC_PATH = '/.well-known/did.json'
+export const DOC_PATH = '/.well-known/did.json'
+
+async function get(url: string, timeout = 5000): Promise<any> {
```

Comment:
tests were failing because this request was just hanging so I added a timeout.

Feel like 5s is pretty reasonable, but lmk if you think it should be different

@pfrazee

Reply:
Yeah that's fine